Source/Tools/FEXLoader/FEXLoader.cpp

```diff
@@ -438,6 +438,15 @@ int main(int argc, char **argv, char **const envp) {
     else {
       Allocator = FEX::HLE::CreatePassthroughAllocator();
     }
+
+    // Now that the upper 32-bit address space is blocked for future allocations,
+    // exhaust all of jemalloc's remaining internal allocations that it reserved before.
+    // TODO: It's unclear how reliably this exhausts those reserves
+    void* data;
+    do {
+      data = malloc(0x1);
+    } while (reinterpret_cast<uintptr_t>(data) >> 32 != 0);
+    free(data);
```

Comment:
Do we have a good way of enabling this only when thunking is enabled? A check for the presence of a thunk config file seems pointlessly generic, whereas a concrete check for enabled thunks would require peeking into ThunkDB internals AFAIK.

Not sure to what extent it matters overall, since the `malloc(1)` spam will generally hit jemalloc's fast path and hence return quickly.